operator/src/test/java/oracle/kubernetes/operator/http/HttpAsyncRequestStepTest.java

```diff
@@ -118,6 +126,19 @@ public void whenErrorResponseReceived_logMessage() {
     assertThat(logRecords, containsFine(HTTP_METHOD_FAILED));
   }
 
+  @Test
+  public void whenThrowableResponseReceived_logMessage() {
+    Memento m = TestUtils.silenceOperatorLogger()
+        .collectLogMessages(logRecords, HTTP_REQUEST_GOT_THROWABLE)
+        .withLogLevel(Level.WARNING)
+        .ignoringLoggedExceptions(HttpAsyncRequestStep.HttpTimeoutException.class);
+    final NextAction nextAction = requestStep.apply(packet);
+
+    completeWithThrowableBeforeTimeout(nextAction, new Throwable("Test"));
+
+    assertThat(logRecords, containsWarning(HTTP_REQUEST_GOT_THROWABLE));
+    m.revert();
```

Comment:
All mementos should be reverted in the `tearDown` method; among other things, that ensures that if the test exceptions out, the revert still happens, so other tests are not affected. It appears that you want to collect an additional log messages. The simplest thing would be simply to add it to the original least created during the `setUp` call. If for some reason you cannot do that, take advantage of the fact that the memento itself is saved as `consoleMemento` and just call its `collectLogMessages` method, here.

Reply:
I started with adding this new memento to the list in setUp method, but it did not work. Let me try consoleMemento suggestion.